I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 36, in script
    jump start_dream #file argue
ScriptError: could not find label 'start_dream'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 36, in script
    jump start_dream #file argue
  File "D:\MY DATA\ren'py\renpy-7.1.3-sdk\renpy\ast.py", line 1633, in execute
    rv = renpy.game.script.lookup(target)
  File "D:\MY DATA\ren'py\renpy-7.1.3-sdk\renpy\script.py", line 875, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start_dream'.

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
Blury 1.0
Tue Dec 25 15:22:40 2018
